pkgs/development/libraries/wayland/default.nix

```diff
@@ -123,4 +123,9 @@ stdenv.mkDerivation rec {
   };
 
   passthru.version = version;
-}
+} // lib.optionalAttrs stdenv.hostPlatform.isStatic {
```

Comment:
usually this would be done with:
`prepatch = lib.optionalString stdenv.hostPlatform.isStatic ''shell code"` instead of using `//`.
With `//` is someone adds a prePatch phase at line 34, it would be silently ignored.

Also is this way of disabling tests specific of static compilation or all cross ?

Reply:
Fixed.
No, this way isn't specific to static compilation thus the nix conditionnal otherwise it would also disable tests on non-static builds.